metadata.json

```diff
@@ -99,12 +99,12 @@
     },
     {
       "name": "puppet",
-      "version_requirement": "3.x"
+      "version_requirement": "3.5"
```

Comment:
3.6 is the oldest place I'm using this module, so I'd be in favor of just going to 3.6 as the oldest Puppet version.  We'll just version this module accordingly.  The validate_cmd seems to rely only on the newer concat version unless I'm missing something.